netsim/augment/groups.py

```diff
@@ -264,11 +281,16 @@ def add_node_level_groups(topology: Box) -> None:
     must_be_list(n,'group',f'nodes.{name}')
 
     for grpname in n.group:
-      if not grpname in topology.groups:
-        topology.groups[grpname] = { 'members': [] }      # Create an empty new group if needed
+      # Sanity check for node groups
+      if topology.groups[grpname].get('type','node') != 'node':
```

Comment:
No need to check if ```grpname``` exists?

Reply:
No, that's a way to create a group (for example, to have one in Ansible inventory) straight from node data.